webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala

```diff
@@ -694,16 +790,19 @@ object TechniqueApi extends Enum[TechniqueApi] with ApiModuleProvider[TechniqueA
     val z: Int = implicitly[Line].value
     val description    = "Get all methods metadata"
     val (action, path) = GET / "methods"
+    val authz: List[AuthorizationType] = AuthorizationType.Technique.Read :: Nil
   }
   case object UpdateMethods            extends TechniqueApiPub with ZeroParam with StartsAtVersion14 with SortIndex {
     val z: Int = implicitly[Line].value
     val description    = "reload methods metadata from file system"
     val (action, path) = POST / "methods" / "reload"
+    val authz: List[AuthorizationType] = AuthorizationType.Technique.Write :: AuthorizationType.Technique.Edit :: Nil
   }
   case object CheckTechnique           extends TechniqueApiPub with ZeroParam with StartsAtVersion16 with SortIndex {
     val z: Int = implicitly[Line].value
     val description    = "Check if a techniques is valid yaml, with rudderc compilation, with various output (json ? yaml ?)"
     val (action, path) = POST / "techniques" / "check"
+    val authz: List[AuthorizationType] = AuthorizationType.Technique.Write :: AuthorizationType.Technique.Edit :: Nil
   }
```

Comment:
I'm not sure if it's just a check (in which case maybe it should be `Read`), or it does something else behind...
I saw it being used in Elm technique editor, shouldn't we allow them to just check the yaml ?